R/RunDiagnostics.R

```diff
@@ -147,28 +174,12 @@ runCohortDiagnostics <- function(packageName = NULL,
     takepackageDependencySnapshot() %>%
     RJSONIO::toJSON(digits = 23, pretty = TRUE)
 
-
   exportFolder <- normalizePath(exportFolder, mustWork = FALSE)
   incrementalFolder <- normalizePath(incrementalFolder, mustWork = FALSE)
 
-  if (!is.null(inclusionStatisticsFolder)) {
-    inclusionStatisticsFolder <- normalizePath(inclusionStatisticsFolder, mustWork = FALSE)
-  }
-
-  if (!is.null(cohortSetReference)) {
-    ParallelLogger::logInfo("Found cohortSetReference. Cohort Diagnostics is running in WebApi mode.")
-    cohortToCreateFile <- NULL
-  }
-  
   start <- Sys.time()
   ParallelLogger::logInfo("Run Cohort Diagnostics started at ", start)
-  
-  if (!is.null(oracleTempSchema) & is.null(tempEmulationSchema)) {
-    tempEmulationSchema <- oracleTempSchema
-    warning('OracleTempSchema has been deprecated by DatabaseConnector')
-  }
-  
-  databaseId <- as.character(databaseId)
```

Comment:
The reason why this is forced to be character is because in one network study a databaseId of value integer was provided - that caused problems when readr was guess data types

this maybe less of an issue now - because we are planning to check conformance to results data model